src/Components/articles-page.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react"
+import { useEffect } from "react"
+import { getArticles } from "../../api"
+import ArticlesList from "./articles-list"
+
+function ArticlesPage(){
+
+    const [articles, setArticles] = useState([])
+
+    useEffect(()=>{
+        getArticles()
+        .then((response)=>{
+            setArticles(response.articles)
+            
+        })
+    }, [])
```

Comment:
Also, what if this request took a long time? Some kind of loading state and message would be great for the user feedback, and avoid us second guessing if the page is broken or not in the case of it taking a while for a sleeping server to spin up